tests/dependency-tests/root.rs

```diff
@@ -88,20 +68,49 @@ fn check_feature_dependency(requirement: Requirement) {
                 i + 1,
                 requirement.search_terms.len()
             );
+            any_failed = true;
         }
     }
+
+    assert!(!any_failed);
+}
+
+fn get_all_wgpu_features() -> Vec<String> {
+    let metadata = cargo_metadata::MetadataCommand::new()
+        .no_deps()
+        .exec()
+        .unwrap();
+
+    metadata
+        .packages
+        .iter()
+        .find_map(|p| {
+            if p.name == "wgpu" {
+                Some(p.features.keys().cloned().collect())
+            } else {
+                None
+            }
+        })
+        .unwrap()
 }
 
 #[test]
-fn wasm32_without_webgl_does_not_depend_on_wgpu_core() {
-    let features_no_webgl: Vec<&str> = ALL_WGPU_FEATURES
+fn wasm32_without_webgl_or_noop_does_not_depend_on_wgpu_core() {
+    let all_features = get_all_wgpu_features();
+
+    let removed_features = ["webgl", "noop", "wgpu-core"];
+
+    let features_no_webgl: Vec<&str> = all_features
         .iter()
-        .copied()
-        .filter(|feature| *feature != "webgl")
+        .map(String::as_str)
+        .filter(|&feature| !removed_features.contains(&feature))
         .collect();
 
+    dbg!(&features_no_webgl);
```

Comment:
nitpick: Gotta 'nother `dbg!(…)` here, was that on purpose?

Reply:
Bleh, no